package.json

```diff
@@ -5,8 +5,8 @@
     "lint": "eslint --fix src/*"
   },
   "dependencies": {
-    "@imstar15/api-augment": "^0.0.12",
-    "@imstar15/types": "^0.0.12",
+    "@imstar15/api-augment": "^0.0.14",
+    "@imstar15/types": "^0.0.14",
```

Comment:
Let’s publish the new code of @oak-network/api-augment and reference it here.

Reply:
to replace your personal library

Reply:
Looks good other than using the personal lib here.

Reply:
Fixed